page-optimize.php

```diff
@@ -272,6 +272,12 @@ function page_optimize_bail() {
 		return true;
 	}
 
+	// Bail if we're in the editor
+	global $pagenow;
+	if ( isset( $pagenow ) && ( $pagenow == 'post.php' || $pagenow == 'post-new.php' ) ) {
```

Comment:
Can we also include the site editor as well?

Reply:
@rcrdortiz You can also check [`WP_Screen::is_block_editor`](https://developer.wordpress.org/reference/classes/wp_screen/is_block_editor/) which should work for all editors:

```
if ( get_current_screen()->is_block_editor() ) {
  return true;
}
```

Reply:
I initially tried the get_current_screen approach but the result is `null` so we need to check the `$pagenow` instead. I'll create a list of pages to exclude so that we can also exclude the site editor.

Reply:
Done! We now don't load the plugin when on the Site Editor.